js/responsive.js

```diff
@@ -1,17 +1,10 @@
 $( document ).ready(function() {
-    var menu_button = false;
-    $( "#column-one" ).prepend( "<img src='images/menu.png' id='menu' onclick='' />" );     //This adds the menu button
-    $('#menu').click(function(){                                                            //This opens and closes the menu
-        if (menu_button == false){
-            $('#page-navigation').animate({'max-height':'500px'}, 100);
-            menu_button = true;
-        }else{
-            $('#page-navigation').animate({'max-height':'0px'}, 100);
-            menu_button = false;
-        }
+    $( "#column-one" ).prepend( "<img src='images/menu.png' id='menu'/>" );     //This adds the menu button, we can add this as hidden element to all sites, or just responsive templates
+    $('#menu').click(function(){
+        $('#page-navigation').toggleClass("open")
     });
     $("#header-name").click(function(event){                                                //This turns the site name into a clickable link.
-        window.location = "/menu/";                                                         //This specific implementation would not work on a celect.org website                                                      
+        window.location = "/menu/";                                                         //This specific implementation would not work on a celect.org website
```

Comment:
This is something I'm sure we could add as a feature.

We have a toggle for clickable header images. We can choose to always make the h1 text clickable too to bring the user "home" if that's what you were going for.